gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/gravitee-apim-rest-api-standalone-distribution-zip/src/main/assembly/plugin-assembly-standalone.xml

```diff
@@ -30,112 +30,21 @@
     <includeBaseDirectory>true</includeBaseDirectory>
 
     <fileSets>
-        <!-- Copy startup scripts and config files -->
         <fileSet>
-            <directory>../target/distribution/config</directory>
-            <outputDirectory>config</outputDirectory>
+            <directory>../target/gravitee-apim-rest-api-distribution-${project.version}</directory>
```

Comment:
It's a bit painful to have the version in the folder name as it forces us to have 1 IntelliJ config per version. Can we remove it?

Reply:
you're right, I will change it